src/resources/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+find_package(Qt5LinguistTools)
+
+add_library(ownCloudResources SHARED
+    client.qrc
+    core_theme.qrc
+    loadresources.cpp
+)
+
+generate_theme(ownCloudResources OWNCLOUD_SIDEBAR_ICONS)
+
+target_link_libraries(ownCloudResources PUBLIC Qt5::Core)
+
+set_target_properties(ownCloudResources PROPERTIES
+    OUTPUT_NAME "${APPLICATION_EXECUTABLE}Resources"
+    AUTOUIC ON
+    AUTORCC ON
+)
+
+GENERATE_EXPORT_HEADER(ownCloudResources
+        EXPORT_MACRO_NAME OWNCLOUDRESOURCES_EXPORT
+        EXPORT_FILE_NAME owncloudresources.h
+        STATIC_DEFINE OWNCLOUD_BUILT_AS_STATIC
```

Comment:
This define is never used.

Reply:
Its used in the generated export header.